### ISSUE TITLE
dragon not spawning visually 

### ISSUE THREAD

**Issue by Duckhunterone** [BONNePlayground/DragonFights#1]
#### Describe the bug

Dragon not spawning in end islands. Also throwing errors in console:
[16:08:27 ERROR]: Entity threw exception at acidisland_world_the_end:128.0,90.0,-128.0
java.lang.NoSuchFieldError: bS
        at lv.id.bonne.dragonfights.v1_17_R1.entity.BentoBoxEnderDragon.movementTick(BentoBoxEnderDragon.java:391) ~[DragonFights-1.1.0.jar:?]
        at net.minecraft.world.entity.EntityLiving.tick(EntityLiving.java:2882) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.world.entity.EntityInsentient.tick(EntityInsentient.java:393) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.level.WorldServer.entityJoinedWorld(WorldServer.java:1114) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.world.level.World.a(World.java:986) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.level.WorldServer.lambda$tick$7(WorldServer.java:775) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.world.level.entity.EntityTickList.a(EntityTickList.java:43) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.level.WorldServer.doTick(WorldServer.java:756) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1619) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:479) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1475) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1274) ~[patched_1.17.1.jar:git-Paper-255]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-255]
        at java.lang.Thread.run(Thread.java:831) [?:?]

#### Steps to reproduce the behavior

Followed installation instructions as per wiki.

#### Expected behavior

Dragon should spawn

#### Screenshots and videos (Optional)

https://youtu.be/emMWPdJPRT4
##### BentoBox Version (Mandatory)

```
bentobox version
[16:07:01 INFO]: Running PAPER 1.17.1.
[16:07:01 INFO]: BentoBox version: 1.17.2
[16:07:01 INFO]: Database: JSON
[16:07:01 INFO]: Loaded Game Worlds:
[16:07:01 INFO]: acidisland_world (AcidIsland): Overworld, Nether, The End
[16:07:01 INFO]: Loaded Addons:
[16:07:01 INFO]: AcidIsland 1.15.0 (ENABLED)
[16:07:01 INFO]: Bank 1.3.2 (ENABLED)
[16:07:01 INFO]: Biomes 1.14.0 (ENABLED)
[16:07:01 INFO]: Border 3.2.0 (ENABLED)
[16:07:01 INFO]: Challenges 0.8.4 (ENABLED)
[16:07:01 INFO]: Chat 1.1.4 (ENABLED)
[16:07:01 INFO]: ControlPanel 1.13.0 (ENABLED)
[16:07:01 INFO]: DimensionalTrees 1.6.0 (ENABLED)
[16:07:01 INFO]: DragonFights 1.1.0 (ENABLED)
[16:07:01 INFO]: Level 2.7.2 (ENABLED)
[16:07:01 INFO]: Limits 1.17.2 (ENABLED)
[16:07:01 INFO]: MagicCobblestoneGenerator 2.3.0 (ENABLED)
[16:07:01 INFO]: Warps 1.11.1 (ENABLED)
```

#### Plugins (Optional)

```
plugins
[16:07:53 INFO]: Plugins (14): BentoBox, CleanroomGenerator, CMI, CMIEInjector*, CMILib, ExcellentEnchants, FastAsyncWorldEdit (WorldEdit), Multiverse-Core, NexEngine, PlaceholderAPI, RoseStacker, Vault, VoidSpawn, WorldBorderAPI
```
 
### Additional context (Optional)

Im aware that its experimental but not sure if it was something on my end that i may have done wrong.

**Comment by BONNe** [comment]
~Can you share the island height value from the acid island config?~

**Comment by Duckhunterone** [comment]
Didnt change heights so as they are in the config as it was. Island height is 50 sea height is 54

**Comment by BONNe** [comment]
It looks like there is incompatibility with Minecraft 1.17.1.
Working on solving it.

**Comment by Duckhunterone** [comment]
ok brilliant thank you

**Comment by BONNe** [comment]
I published a new 1.2.0 version in releases that allows to run it on 1.1.7.1 servers.

**Comment by Duckhunterone** [comment]
Thank you will test in a bit
